core/tests/bank_forks.rs

```diff
@@ -146,7 +145,7 @@ mod tests {
                 &CompressionType::Bzip2,
             ),
             CompressionType::Bzip2,
-            &GenesisConfig::default(),
+            old_genesis_config,
```

Comment:
This spent some time for me to figure out why the test has started to failing.....